build/Makefile

```diff
@@ -59,6 +59,8 @@ KIND_CONTAINER_NAME=$(KIND_PROFILE)-control-plane
 # Game Server image to use while doing end-to-end tests
 GS_TEST_IMAGE ?= gcr.io/agones-images/udp-server:0.19
 
+FEATURE_GATES ?= "PlayerTracking=true&ContainerPortAllocation=true"
```

Comment:
Maybe this should be `ALL_FEATURE_GATES` so that then for targets that want to default to full feature games one could do:

`target: FEATURE_GATE ?= $(ALL_FEATURE_GATES)`

And it won't conflict with anything it shouldn't.

Reply:
Ok, I will try this solution.

Reply:
@markmandel done. Please check.